test/next/src/app/vercel/blob/app/test/edge/page.tsx

```diff
@@ -0,0 +1,15 @@
+import * as vercelBlob from '@vercel/blob';
+
+export const runtime = 'edge';
+
+export default async function AppBodyEdge(): Promise<JSX.Element> {
+  const blob = await vercelBlob.put('test-app-edge.txt', 'Hello from Pages', {
+    access: 'public',
+  });
+  return (
+    <>
+      <h1>App Router direct body upload example via an Edge Function</h1>
```

Comment:
We're not receiving a body here right, we're just creating a blob immediately once the page loads, this is fine but perhaps we need to change the wording

Reply:
body refers to the type of fie upload...btw as we want to download the content I could show it here

Reply:
> body refers to the type of fie upload

Let's call it direct string upload?

> I could show it here

Indeed